src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Resources/Patch/FhirPatchBuilderTests.cs

```diff
@@ -17,31 +17,35 @@ public void GivenAFhirPatchParameterComponent_WhenOperationInvalid_ThenInvalidOp
         {
             var patchParam = new Parameters().AddPatchParameter("remove", path: "Patient.identifier");
 
-            Assert.Throws<InvalidOperationException>(() => new FhirPathPatchBuilder(new Patient(), patchParam));
+            var exception = Assert.Throws<InvalidOperationException>(() => new FhirPathPatchBuilder(new Patient(), patchParam));
```

Comment:
This is a very complete comparison on the error message.  I wonder if it needs to be quite so complete?  If any part of the error message changes, even just whitespace, then the test would fail.  Perhaps we can assert only a part of the error?

Reply:
Huh - literally have never thought of that. I don't have much experience in this error. Which of the below would be best?
- Using a string contains to look for the most important part of the error messages
- Creating an exception that is based off of `InvalidOperationException`. This would contain the path and the operation. This would be used to create the exception string. The test would check the fields used to build the string vs the string.

Reply:
@feordin I changed these to less restrictive "contains" 😄